common/templates/general.go

```diff
@@ -258,6 +258,10 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 		case "filename":
 			// Cut the filename to a reasonable length if it's too long
 			filename = common.CutStringShort(ToString(val), 64)
+		case "reply":
+			msg.Reference = &discordgo.MessageReference{
+				MessageID: val.(int64),
```

Comment:
```suggestion
				MessageID: ToInt64(val),
```

Consider this change instead, otherwise it will fail with a bare int, like in this code:

```go
{{/* any valid message ID, hardcoded */}}
{{ $id := 1004375006253424680 }}
{{ sendMessage nil ( complexMessage "content" "hello" "reply" $id ) }}
```

Without this suggested change, users will have to cast `toInt64` themselves, which is not immediately obvious.